src/pysdmx/model/submission.py

```diff
@@ -0,0 +1,20 @@
+"""SDMX Submission classes."""
+
+from msgspec import Struct
+
+
+class SubmissionResult(Struct):
```

Comment:
Do we want to make it immutable too?

Reply:
Yes, probably. Should be frozen. I will make the change.